packages/paste-website/src/pages/theme/changing-theme.mdx

```diff
@@ -88,3 +88,7 @@ import {Customization} from '@twilio-paste/core/customization';
 
 </content>
 </contentwrapper>
+
+### How do I change themes in Figma?
+
+If you're using Paste in Figma, you can swap between libraries using a native Figma feature. Find the [theme switching documentation](/theme/dark-theme#theme-switching) for more details.
```

Comment:
todo: Looks like this needs to go above </content>.

Can you also update the last sentence to "Go to the [theme switching documentation](/theme/dark-theme#theme-switching) for more details."?